cmd/wasm-client/go.mod

```diff
@@ -2,18 +2,20 @@ module github.com/lightninglabs/lightning-node-connect/cmd/wasm-client
 
 require (
 	github.com/btcsuite/btcd/btcec/v2 v2.3.4
-	github.com/btcsuite/btclog v0.0.0-20241003133417-09c4e92e319c
+	github.com/btcsuite/btclog/v2 v2.0.1-0.20250110154127-3ae4bf1cb318
 	github.com/jessevdk/go-flags v1.4.0
-	github.com/lightninglabs/faraday v0.2.13-alpha
-	github.com/lightninglabs/lightning-node-connect v0.3.3-alpha.0.20250303090341-f823ad04b9de
-	github.com/lightninglabs/lightning-terminal v0.14.0-alpha
```

Comment:
I'm just a little worried that we'd end up missing protos that's part of `litd` but not `litrpc` in the future, but I think the `looprpc` and `pool` dependencies here should cover those. But we just need to be careful in the future, and ensure we keep all of them updated.